games-engines/renpy/renpy-8.0.3.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
- EAPI 8
- Drop Python 3.8 (it does nothing now)
- Add Python 3.11? See gentoo-dev ML too.